### ISSUE TITLE
The interiors of closed paths are not exported correctly

### ISSUE THREAD

**Issue by glopesdev** [managed-commons/SvgNet#22]
The default `FillMode` for graphics path is `Alternate`, where the interiors of closed paths are automatically excluded to form holes. Unfortunately, this is currently not supported by SvgNet, as demonstrated by the following snippet which can be added to the end of the path sample in `GdiTestForm`:

```c#
GraphicsPath myPath3 = new GraphicsPath();
myPath3.AddEllipse(120, 50, 50, 50);
myPath3.AddEllipse(135, 65, 30, 30);
ig.FillPath(Brushes.Blue, myPath3);
```

![gditest](https://user-images.githubusercontent.com/5315880/55276117-93aca900-52e7-11e9-9ac7-3389d34fa713.jpg)

Looking at the exported SVG code, it looks like the inner path is exported as an independent path, whereas in the SVG specification it may be necessary to merge everything into a single path, as in this example: https://www.w3.org/TR/SVG/images/painting/fillrule-evenodd.svg

**Comment by glopesdev** [comment]
Also the style for `SvgPathElement` instances created by `FillBeziers` needs to be set to use `fill-rule:evenodd`. For some reason this was done for polygons, but not for beziers, even though `FillMode` is one of the parameters for the method (which is never used inside):

```c#
private void FillBeziers(Brush brush, PointF[] points, FillMode fillmode)

...

if (fillmode == FillMode.Alternate)
{
    bez.Style.Set("fill-rule", "evenodd");
} else
{
    bez.Style.Set("fill-rule", "nonzero");
}
```

**Comment by monoman** [comment]
Merged and published as nuget 2.0.1 - still validating but should be listed soon.
